code/game/atoms_movable.dm

```diff
@@ -249,6 +257,9 @@
 //Called after a successful Move(). By this point, we've already moved
 /atom/movable/proc/Moved(atom/OldLoc, Dir, Forced = FALSE)
 	SEND_SIGNAL(src, COMSIG_MOVABLE_MOVED, OldLoc, Dir, Forced)
+	if(OldLoc != loc)
+		loc?.Entered(src, OldLoc)
+		OldLoc?.Exited(src, loc)

```

Comment:
These are called by byond, what are you doing here?

Reply:
You’d think that be the case but entered and exited aren’t byond procs but Enter and Exit are

Reply:
This is the same symptoms I saw when loc wasnt being normalized and step_ vars were being set directly. You shouldn't have to do this and there's some other problem if this is needed.

Reply:
I must've fixed this some other way because removing this still allows space transitions to work now